scripts/go_executable_build.sh

```diff
@@ -33,7 +33,7 @@ esac
 declare -A LIB
 
 if [ "$(uname -s)" == "Darwin" ]; then
-   MD5='md5 -r'
+   MD5=md5sum
```

Comment:
md5 is part of openssl package. It works on my mac with catalina.

Reply:
it is not available on my macos. I did not explicitly changed anything other than upgrading macos to catalina.
I think its better to specify brew install md5sha1sum to install md5sum explicitly, since this could happen to others as well.